docs/js-core-3/week-1/lesson.md

```diff
@@ -431,32 +431,73 @@ The project is called [TV Show](../tv-show-dom-project/readme.md)
 
-:::
+:::note Exercise (15 min)
 
-:::note Exercise (10 minutes)
+**Exercise objective:** To identify the steps to form a constructive response.
 
-**Exercise objective:** To develop user story drafts with detail and tasks
+Imagine that you are at a new job and one colleague has scheduled to collaborate/meet with you several times and then canceled at the last minute. You need to meet with this person to take the next steps in the project that you are both working on. Use the list below to prepare for the conversation you will have with them. Discuss as a class what you would say to your colleague.
 
-Review the stories you have created for Level 100 in your TV Show Project but this time put some details on those stories and break that work down into tasks. Once finished, we will share some of those stories and their tasks as a group.
+- What is the key problem to be addressed?
+- What impact is this having on you/the team/work?
+- What responsibility are you going to take for your part?
+- What do you want to achieve by the end of the conversation?
 
 :::
 
+:::note Exercise (35 min)
+
+**Exercise objective:** To role-play a difficult conversation.To role-play a difficult conversation.
+
+Work in pairs. Use one of the following scenarios where you need to have a difficult conversation. If you finish your first scenario and still have time, choose another one.
+
+Activities you will do are listed below:
+
+Role-play the conversation with your partner. List emotions that you associate with that moment whilst doing the activity (10 min)
+
+Discuss with your partner how that conversation could have been better if you were able to take the emotion out of it (5 min)
+
+Role-play scenarios:
+
+1. You are a developer in a team. A very important project has been presented; however, you have not been assigned to it. The person who will be working on this project is a peer, who you think isn’t as competent as you. You want to question that decision with your manager.
+2. You are working on a team with a person you have had issues with before. You think that they are not doing their share of the work on purpose because they dislike you. You want to talk to this person about it.
+3. You have been working hard and would like to get a pay rise. Have this discussion with your manager. 
+4. You think that the project you have been allocated to is impossible and it can't succeed and you are worried you are being set up to fail. You want to talk to your Product Manager or Product Owner about this.
+5. Someone in your team isn’t doing their share of the work and you want to talk to them about it.
```

Comment:
Is this meaningfully different from 2? If so, it may be worth calling out the focus of this one?

Reply:
I think they are different because no.2 is about personal relationship dynamics ("You think that they are not doing their share of the work on purpose because they dislike you.")

Reply:
I have amended the sentence with your suggestion.